### DISCUSSION TITLE
Is it possible to update the current date?

### DISCUSSION THREAD

**Discussion by vaio97** [vkurko/calendar#181] [Q&A]
If I keep the calendar open for more than one day, the current day is always displayed as the opening day without ever being updated. Should the library automatically update it or is there a method to change the current date (move the 'ec-today' class to the next day)?

**Comment by vkurko** [comment]
Hello. It should work in `timeGrid` views but it looks like it does not in the others. I'll fix it in the next release.

**Comment by vkurko** [comment]
@vaio97 The issue should be fixed in v2.3.3. Please check.
